### PR TITLE
flipping error bands

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

**Comment by andrea-pasquale** [comment]
Maybe it could be useful to restore a flipping with signal `flipping_signal`

File: src/qibocal/protocols/characterization/flipping_signal.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            else None
+        )
+        f = flips[index] / (flips[1] - flips[0]) if index is not None else 1
+        y_max = np.max(voltages)
+        x_min = np.min(flips)
+        x_max = np.max(flips)
+        x = (flips - x_min) * 2 * np.pi * f / (x_max - x_min)
+        y = (voltages - y_min) / (y_max - y_min)
+
+        pguess = [0.5, 0.5, 1, np.pi, 0]
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/flipping_signal.py:155]: This error was already in main but `pguess` should include `f`

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @Edoardo-Pedicillo.
Please find below some suggestions.

**Comment by andrea-pasquale** [comment]
It seems to be working as expected:
http://login.qrccluster.com:9000/4BN4XN_wRhiYuA0G_IV2NQ==/
http://login.qrccluster.com:9000/SlDyacO2Rt6QO4TjAIqp5A==/
Thanks @Edoardo-Pedicillo, could you please add also the $\chi^2$ for the `flipping` protocol?

**Review by andrea-pasquale — APPROVED** [review]

File: src/qibocal/protocols/characterization/flipping_signal.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+@dataclass
+class FlippingSignalData(FlippingData):
+    """Flipping acquisition outputs."""
+
+    data: dict[QubitId, npt.NDArray[FlippingType]] = field(default_factory=dict)
+    """Raw data acquired."""
+
+
+def _acquisition(
```
Comment (Jacfomg) [inline src/qibocal/protocols/characterization/flipping_signal.py:46]: Just as a comment if we are gonna have several routines like this, where it is the same as another just chaging the `exection_parameters`  should we find a way of not repeating the whole acquisition to reduce code duplication ?

**Review by Jacfomg — APPROVED** [review]

**Comment by andrea-pasquale** [comment]
Coverage is failing but it will be probably be fixed by #707. Let's merge.

**Pull request merged**